scdl/scdl.py

```diff
@@ -450,6 +457,12 @@ def download_track(track, playlist_name=None, playlist_file=None):
                 logger.debug(e)
         else:
             logger.error("This type of audio doesn't support tagging...")
+            
```

Comment:
Oh! My bad. It's totally unprofessional.
My first and last time I submit patch directly with the github editor code.
Specially when it's late in the night >_<